modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/SlaCard/index.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import classNames from 'classnames';
+import {useEffect, useState} from 'react';
+import client from '../../../../apolloClient';
+import {getKoroneikiAccounts} from '../../../../common/services/liferay/graphql/queries';
+import getSlaNames from '../../../../common/utils/getSlaNames';
+import SlaCardLayout from './Inputs';
+
+const SlaCard = ({accountKey}) => {
+	const [slaData, setSlaData] = useState();
+	const [slaSelected, setSlaSelected] = useState('');
+
+	useEffect(() => {
+		const fetchkoroneikeAccount = async () => {
+			const {data} = await client.query({
+				query: getKoroneikiAccounts,
+				variables: {
+					filter: `accountKey eq '${accountKey}'`,
+				},
+			});
+			if (data) {
+				const dataPath = data.c.koroneikiAccounts.items[0];
+				const formatDate = {
+					day: '2-digit',
+					month: '2-digit',
+					year: 'numeric',
+				};
+				const slaFiltedData = [];
+				const slaName = getSlaNames();
+				const slaPathLabel = {
+					current: slaName.current.toUpperCase(),
+					expired: slaName.expired.toUpperCase(),
+					future: slaName.future.toUpperCase(),
+				};
+
+				const slaNewDate = (sla) => {
+					return new Date(dataPath[sla]).toLocaleDateString(
+						'en-US',
+						formatDate
+					);
+				};
+				const slaGetData = (sla) => {
+					const slaFullTitle = 'sla' + sla;
+
+					return {
+						slaDateEnd: `${slaNewDate(slaFullTitle + 'EndDate')}`,
+						slaDateStart: `${slaNewDate(
+							slaFullTitle + 'StartDate'
+						)}`,
+						slaLabel: sla.toUpperCase(),
+						slaTitle: dataPath[slaFullTitle].split(' ')[0],
+					};
+				};
+
+				const slaRawData = [
+					slaGetData(slaName.current),
+					slaGetData(slaName.expired),
+					slaGetData(slaName.future),
+				].filter((sla) => sla.slaTitle);
+				const slaRaw = {
+					first: slaRawData[0],
+					second: slaRawData[1],
+					third: slaRawData[2],
```

Comment:
Who is first second and third? Don't you think it would be better a more detailed name?

Reply:
OK. Done!